examples/missing_values.py

```diff
@@ -1,50 +1,65 @@
 """
-======================================================
+====================================================
 Imputing missing values before building an estimator
-======================================================
+====================================================
 
-This example shows that imputing the missing values can give better results
-than discarding the samples containing any missing value.
-Imputing does not always improve the predictions, so please check via cross-validation.
+This example shows that imputing the missing values can give
+better results than discarding the samples containing any missing value.
+Imputing does not always improve the predictions,
+so please check via cross-validation.
 Sometimes dropping rows or using marker values is more effective.
 
 Missing values can be replaced by the mean, the median or the most frequent
 value using the ``strategy`` hyper-parameter.
 The median is a more robust estimator for data with high magnitude variables
 which could dominate results (otherwise known as a 'long tail').
 
-Script output::
+Another option is the MICE imputer. This uses round-robin linear regression,
+treating every variable as an output in turn. The simple version implemented
+assumes Gaussian output variables. If your output variables are obviously
+non-Gaussian, consider transforming them for improve performance.
 
-  Score with the entire dataset = 0.56
-  Score without the samples containing missing values = 0.48
-  Score after imputation of the missing values = 0.55
+Script output:
+
+  MSE with the entire dataset = 3354.15
+  MSE without the samples containing missing values = 2968.98
+  MSE after mean imputation of the missing values = 3507.77
+  MSE after MICE imputation of the missing values = 3340.39
```

Comment:
Here the MSE is better than "MSE with the entire dataset", and better than "MSE after mean imputation of the missing values". Were you hoping for a more dramatic improvement?